### PR TITLE
Rename charm configurations to include new prefixes

### PR DESCRIPTION
**Pull request by addyess**
Applicable spec: KU-2098

### Overview

Add `bootstrap-` and `node-` prefixes to charm config

### Rationale

Provides better namespacing to charm options

### Library Changes

Needs updated library changes in 
* https://github.com/charmed-kubernetes/layer-kubernetes-node-base/pull/12

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)

### CODE REVIEW

File: charms/worker/k8s/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 charm-lib-contextual-status @ git+https://github.com/charmed-kubernetes/charm-lib-contextual-status@255dd4a23defc16dcdac832306e5f460a0f1200c
 charm-lib-interface-external-cloud-provider @ git+https://github.com/charmed-kubernetes/charm-lib-interface-external-cloud-provider@e1c5fc69e98100a7d43c0ad5a7969bba1ecbcd40
-charm-lib-node-base @ git+https://github.com/charmed-kubernetes/layer-kubernetes-node-base@9b212854e768f13c26cc907bed51444e97e51b50#subdirectory=ops
+charm-lib-node-base @ git+https://github.com/charmed-kubernetes/layer-kubernetes-node-base@KU-2098/label-maker-options#subdirectory=ops
```
Comment (addyess) [inline charms/worker/k8s/requirements.txt:3]: DO NOT MERGE UNTIL THIS comment is resolved.  The node-base library cannot reference a feature branch

File: charms/worker/k8s/src/charm.py [hunk 2]
```diff
@@ -322,8 +326,9 @@ def _bootstrap_k8s_snap(self):
         bootstrap_config = BootstrapConfig.construct()
         self._configure_datastore(bootstrap_config)
         bootstrap_config.cluster_config = self._assemble_cluster_config()
-        bootstrap_config.service_cidr = str(self.config["service-cidr"])
-        bootstrap_config.control_plane_taints = str(self.config["register-with-taints"]).split()
+        bootstrap_config.service_cidr = str(self.config["bootstrap-service-cidr"])
+        bootstrap_config.pod_cidr = str(self.config["bootstrap-pod-cidr"])
+        bootstrap_config.control_plane_taints = str(self.config["bootstrap-node-taints"]).split()
```
Comment (HomayoonAlimohammadi) [inline charms/worker/k8s/src/charm.py:331]: Do we need to rename this attr? Are these taints only related to control plane nodes?

**Review by bschimke95 — APPROVED** [review]
LGTM

File: charms/worker/k8s/src/charm.py (hunk 2 above)
Comment (addyess) [reply charms/worker/k8s/src/charm.py:331]: on bootstrap, yes they're only related to control-planes

https://github.com/canonical/k8s-snap-api/blob/main/api/v1/type_bootstrap_config.go#L14

Comment (HomayoonAlimohammadi) [reply charms/worker/k8s/src/charm.py:331]: Correct, thanks a lot!
